build.gradle.kts

```diff
@@ -16,6 +16,10 @@ dependencies {
   runtimeOnly("org.flywaydb:flyway-core:6.5.6")
 }
 
+tasks.register("fix") {
+  dependsOn(":ktlintFormat")
+}
+
```

Comment:
./gradlew fix

to fix any ktlint problems

Reply:
If you run `./gradlew addKtlintFormatGitPreCommitHook` it adds a git pre commit hook that runs `ktlintFormat` for you.
There's another one - `./gradlew ktlintApplyToIdeaGlobally` which aligns the Intellij formatting rules with ktlint.

Reply:
Which is mentioned right at the very end of the Gradle plugin readme... https://github.com/ministryofjustice/dps-gradle-spring-boot#code-style--formatting

Hmmm I guess that's not very easy to find :thinking: